params/config.go

```diff
@@ -372,6 +393,9 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 		{name: "petersburgBlock", block: c.PetersburgBlock},
 		{name: "istanbulBlock", block: c.IstanbulBlock},
 		{name: "muirGlacierBlock", block: c.MuirGlacierBlock, optional: true},
+		{name: "berlinBlock", block: c.BerlinBlock},
+		{name: "londonBlock", block: c.LondonBlock},
+		{name: "shanghaiTime", timestamp: c.ShanghaiTime},
```

Comment:
just adding the items we'd need to switch to libevm, not everything needs to be added back at once.